.changeset/breezy-wolves-joke.md

```diff
@@ -0,0 +1,10 @@
+---
+'sku': patch
+---
+
+Allow `:` to be used in dynamic paths again.
+
+Previously, dynamic paths were declared using the standard `:param` syntax, but this had been deprecated in favour of `$`.
+
+This has now been updated to allow for both.
+This should allow `sku serve` to work for projects using colon syntax.
```

Comment:
Feel like this is a bit misleading.`:` still works, and always has, but now sku serve supports it, instead of throwing an error.

Reply:
[The documentation](https://seek-oss.github.io/sku/#/./docs/static-rendering?id=dynamic-routes) is pretty clear that the syntax is `$`. It might have worked by accident before, but I don't think there's anything wrong with saying "this is now expressly permitted".